src/test/java/org/opensearch/neuralsearch/query/HybridQueryExplainIT.java

```diff
@@ -703,6 +668,154 @@ public void testExplain_whenRRFProcessor_thenSuccessful() {
         assertTrue((double) explanationsHit4.get("value") > 0.0f);
     }
 
+    @SneakyThrows
+    public void testExplain_whenMinMaxNormalizationWithLowerBounds_thenSuccessful() {
```

Comment:
Shall we also add some edge cases in IT, like case: lower bound greater than max score?

Reply:
added test case to org.opensearch.neuralsearch.processor.NormalizationProcessorIT